yarn.lock

```diff
@@ -7036,6 +7036,24 @@ expo-cli@^3.11.1:
     "@expo/traveling-fastlane-darwin" "1.11.0"
     "@expo/traveling-fastlane-linux" "1.11.0"
 
+expo-module-scripts@~1.2.0:
```

Comment:
I'm not sure but this change in `yarn.lock` shouldn't be there. If I recall correctly, this means something installed `expo-module-scripts` from outside of the workspace. Could you check whether the version of `expo-module-scripts` in the workspace matches the semver pattern that we use in the packages you changed?

Reply:
You might be right. It disappeared after yet another `yarn`, so probably there was some misalignment in versions.